RLBotDiscordBot/bot.py

```diff
@@ -46,15 +48,19 @@ def __init__(self):
     async def on_ready(self):
         self.logger.info(f'{self.user} online! (ID: {self.user.id})')
         self.has_reacted = 0
-        self.has_checked = False
 
     async def on_message(self, message):
+
         if not self.has_checked:
             for member in message.guild.members:
                 y = member.roles
                 for role in y:
                     if role.name == "Python" or role.name == "Java" or role.name == "C#" or role.name == "Rust" or role.name == "Scratch" or role.name == "C++":
```

Comment:
add a way to add more roles to this list. our language library is always growing after all :)

Reply:
```suggestion
                    if role.name in ["Python" , "Java", "C#", "Rust", "Scratch", "C++"]:
```
Then get that array from the config (including adding / removing using commands)